src/elements/Segment/SegmentGroup.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+import _Segment from './_Segment'
+
+function SegmentGroup(props) {
+  const {
+    children, className,
+  } = props
+
+  const classes = cx(
+    className
+  )
+
+  const rest = getUnhandledProps(SegmentGroup, props)
+
+  return (
+    <_Segment className={classes} {...rest} _sdClass='sd-segment-group' _segmentClass='segments'>
+      {children}
+    </_Segment>
```

Comment:
On another note on your first point, I've pretty much been going on the Stardust docs for figuring out props for each component.  These docs say Segment can be raised, piled, stacked etc.  I guess these examples need relocated under Segments?